src/client/game.cpp

```diff
@@ -3245,7 +3245,7 @@ void Game::processPlayerInteraction(f32 dtime, bool show_hud)
 	const ItemStack &tool_item = player->getWieldedItem(&selected_item, &hand_item);
 
 	const ItemDefinition &selected_def = selected_item.getDefinition(itemdef_manager);
```

Comment:
```suggestion
```
Looks to me like you don't need this temporary anymore.

Reply:
I actually did remove it. If you go down further into this function, the `selected_def` reference is actually used. Should it be moved down to where it is used?

Reply:
Yes.